src/internal/connector/sharepoint/data_collections.go

```diff
@@ -117,6 +117,7 @@ func DataCollections(
 	if len(collections) > 0 {
 		baseCols, err := graph.BaseCollections(
 			ctx,
+			nil,
```

Comment:
I would feel better if we passed in `collections` here as expected instead of nil
```suggestion
			collections,
```

Reply:
oh good catch, thanks.